lib/record_store/record/txt.rb

```diff
@@ -10,16 +10,12 @@ def initialize(record)
       @txtdata = record.fetch(:txtdata)
     end
 
-    def to_s
-      "[#{type}Record] #{fqdn} #{ttl} IN #{type} \"#{rdata_txt}\""
-    end
-
     def rdata
       { txtdata: txtdata }
     end
 
     def rdata_txt
-      txtdata
+      %("#{txtdata.gsub('"', '\"')}")
```

Comment:
This logic is repeated in the `Record#build_from_api`. Do we still need this?

Reply:
`build_from_api` is unescaping
this is escaping